src/app/GitUI/WindowsJumpListManager.cs

```diff
@@ -107,6 +107,7 @@ public void AddToRecent(string workingDir)
 
                 // sanitise
                 StringBuilder sb = new(repositoryDescription);
+                sb.Replace(@":\", "_");
```

Comment:
discussion: 
It would be nize to have / instead of _ in the name, but this is a filename
Similar, I would like to replace \\wsl$ with /wsl or so but very minor diff anyway